<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>site Gestion de stock d’un magasin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	border-image: url(:/images/img2.jpg);

}</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 255, 203);</string>
    </property>
    <property name="text">
     <string>Add Produit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Fournisseur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Vérifier Stock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Gérer Sortie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Gérer l'entre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>List</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>621</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(231, 231, 255);
background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-image: url(:/images/img3.gif);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Criteria</string>
    </property>
    <widget class="QRadioButton" name="rdoProduit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(231, 231, 255);</string>
     </property>
     <property name="text">
      <string>Produit</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoFournisseur">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(231, 231, 255);</string>
     </property>
     <property name="text">
      <string>Fournisseur</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoClient">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(231, 231, 255);</string>
     </property>
     <property name="text">
      <string>Client</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
